rdmo/management/tests/helpers_import_elements.py

```diff
@@ -72,7 +73,13 @@ def _test_helper_change_fields_elements(elements,
     return _new_elements
 
 
-def parse_xml_and_import_elements(xml_file):
+def parse_xml_and_import_elements(xml_file, shuffle_elements=False):
     elements, root = read_xml_and_parse_to_root_and_elements(xml_file)
+    if shuffle_elements:
```

Comment:
It can shuffle the elements before the import (with save). So that it can be tested that even when elements arrive in random order that the related items are also properly saved (because the sorting algorithm functioned properly).
I think I forgot to make a parametrized test for it that uses/not-uses this shuffling.